modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,86 @@
+    region: us-east1
+    tags: <10>
+      - global-tag1
+controlPlane:
+  hyperthreading: Enabled <11>
+  name: master
+  platform:
+    gcp:
+      tags: <10>
+      - control-plane-tag1
+      type: n2-standard-4
+      zones:
+      - us-central1-a
+      - us-central1-c
+  replicas: 3
+compute:
+- hyperthreading: Enabled <11>
+  name: worker
+  platform:
+    gcp:
+      tags: <10>
+      - control-plane-tag1 
+      type: n2-standard-4
+      zones:
+      - us-central1-a
+      - us-central1-c
+  replicas: 3
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+pullSecret: '{"auths": ...}'
+fips: false <12>
+sshKey: ssh-ed25519 AAAA... <13>
+----
+<1> `credentialsMode` must be set to Passthrough to allow the cluster to use the provided GCP service account after cluster creation.
+<2> The name of the subnet in the shared VPC for compute machines to use.
+<3> The name of the subnet in the shared VPC for control plane machines to use.
+<4> Optional. If you set `createFirewallRules` to "Disabled", you can create and manage firewall rules manually through the use of network tags. By default, the cluster will automatically create and manage the firewall rules that are required for cluster communication. Your service account must have `compute.NetworkAdmin` and `compute.securityAdmin` privileges in the host project to perform these tasks automatically. 
+<5> The name of the shared VPC.
+<6> The name of the host project where the shared VPC exists.
+<7> Optional. The name of a public DNS zone in the host project. Your service account must have the `dns.admin` privilege in the host project. If you do not set this value, the installation program will use the public DNS zone in the service project.
+<8> Optional. The name of the host project which contains the public DNS zone. This value is required if you specify a public DNS zone that exists in another project.
+<9> The name of the GCP project where you want to install the cluster.
+<10> Optional. If you want to manually create and manage your GCP firewall rules, you can set `platform.gcp.createFirewallRules` to "Disabled" and then specify one or more network tags. You can set tags to apply to the compute machines, the control plane machines, or all machines.
+<11> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
+<12> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
```

Comment:
hypertthreading and fips aren't directly related to the Shared VPC workflow. I'm not sure we need to specifically call them out here as they are covered later in the `Optional configuration parameters` section.